app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventCapture/EventCapturePresenterImpl.java

```diff
@@ -552,8 +538,9 @@ public void onNextSection() {
 
     private void changeSection() {
 
-        if (!errors.isEmpty() && errors.get(currentSection.get()) != null)
-            return;
+        if (!errors.isEmpty() && errors.get(currentSection.get()) != null) {
+            view.showSnackBar(R.string.fix_error);
```

Comment:
can you remove this dependency from the presenter pls (R.string.fix_error)

Reply:
Of course!

Reply:
done!